README.md

```diff
@@ -125,6 +125,16 @@ cd torch2trt
 sudo python setup.py install --plugins
 ```
 
+### Option 3 - With experimental support for Quantization Aware Training
```

Comment:
Here's the wording I think I currently prefer

```md
### Option 3 - With support for experimental community contributed features

To install torch2trt with experimental community contributed features under ``torch2trt.contrib``, like Quantization Aware Training (QAT), call the following, 

...

This enables you to run the QAT example located [here](examples/contrib/quantization_aware_training).
```

Let me know if you see any issues.